### PR TITLE
docs(CONTRIBUTING.md): added dvc subsection

### PR DESCRIPTION
**Pull request by leogail**
non optimal pipeline for now (look at first point and 'remarque') but non-breaking with git architecture

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -305,6 +305,46 @@ $ npm run insert:description services/terms-extraction/v*.md
...
+    - entrer le login : `dvc remote modify --local webdav-remote login YOUR_LOGIN`
+    - entrer le mot de passe : `dvc remote modify --local webdav-remote password YOUR_PASSWORD`
+  - push le fichier sur le webdav : `dvc add DOSSIER_OU_FICHIER_A_PUSH` puis `dvc push`, sans se soucier du nom
+  - Le fichier `DOSSIER_OU_FICHIER_A_PUSH.dvc` est créé et devra être copié à l'endroit où le modèle doit être dans le code
+  - ***remarque** : il est possible de faire tout ça dans le repo git directement, cela ajoutera simplement un gitignore qu'il ne faudra pas déplacer ou supprimer. Aussi, les dossiers .dvc et .dvcignore ne seront ni à ignorer par git, ni a push sur le repo*
+- Créer un fichier .env à la racine **du service** (./services/\<service-name\>/.env) qui ressemblera à ça :
+
+  ```bash
+  #!/usr/bin/env bash
+
```
Comment (parmentf) [inline CONTRIBUTING.md:327]: **polish:** Remove shebang

It is not useful in a `.env` that you import via `. .env`, on the contrary to a `env.bash` that you run via `./env.bash`.

```suggestion
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -305,6 +305,46 @@ $ npm run insert:description services/terms-extraction/v*.md
...
 > fichiers, utilisez plutôt `./bin/insert-description.sh`.
 
+### Utilisation de DVC (pour charger des données ou des modèles)
+
+DVC est un outil de versionnage de données. Lorsqu'on créé un service qui nécessite un modèle ou un fichier, il est nécessaire de l'utiliser pour ne pas avoir de gros fichiers sur git.
+En plus du reste, il faut suivre ces étapes lorsqu'on utilise DVC :
+
+- S'assurer d'avoir déposé les données sur le webdav du service TDM en ayant préalablement utilisé DVC (pour cela : )
+  - mettre son fichier nommé `DOSSIER_OU_FICHIER_A_PUSH` dans un autre dossier.
+  - Initier un dépot DVC en faisant `git init` puis `dvc init`.
```
Comment (parmentf) [inline CONTRIBUTING.md:315]: **thought (non-blocking):** ‏Only if not already in a git repo.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -305,6 +305,46 @@ $ npm run insert:description services/terms-extraction/v*.md
...
+
+  ```bash
+  #!/usr/bin/env bash
+
+  export WEBDAV_URL=webdavs://YOUR_WEBDAV_URL.fr
+  export WEBDAV_LOGIN=YOUR_LOGIN
+  export WEBDAV_PASSWORD=YOUR_PASSWORD 
+  ```
+
+- modifier les scripts `build:dev` et `build` du fichier `package.json`
```
Comment (parmentf) [inline CONTRIBUTING.md:333]: **quibble (non-blocking):** of the service
‏
```suggestion
- modifier les scripts `build:dev` et `build` du fichier `package.json` du service
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -305,6 +305,46 @@ $ npm run insert:description services/terms-extraction/v*.md
...
+    ". ./.env 2> /dev/null; DOCKER_BUILDKIT=1 docker build -t cnrsinist/${npm_package_name}:latest --secret id=webdav_login,env=WEBDAV_LOGIN --secret id=webdav_password,env=WEBDAV_PASSWORD --secret id=webdav_url,env=WEBDAV_URL ." 
+    ```
+
+  - pour `build`:
+
+    ```txt
+     ". ./.env 2> /dev/null; DOCKER_BUILDKIT=1 docker build -t cnrsinist/${npm_package_name}:${npm_package_version} --secret id=webdav_login,env=WEBDAV_LOGIN --secret id=webdav_password,env=WEBDAV_PASSWORD --secret id=webdav_url,env=WEBDAV_URL ."
+     ```
+
+- modifier le dockerfile en conséquent, en s'inspirant du dockerfile de `biblio-ref`
```
Comment (parmentf) [inline CONTRIBUTING.md:346]:
```suggestion
- modifier le `Dockerfile` en conséquence, en s'inspirant du [Dockerfile de `biblio-ref`](https://github.com/Inist-CNRS/web-services/blob/main/services/biblio-ref/Dockerfile)
```

**Review by parmentf — CHANGES_REQUESTED** [review]
Few improvements to do.

File: CONTRIBUTING.md (hunk 2 above)
Comment (leogail) [reply CONTRIBUTING.md]: modifié : Initier un dépot DVC en faisant `dvc init` (nécessite d'être dans un repo git).

**Pull request merged**